include/commissioning/commissioner.h

```diff
@@ -35,6 +35,16 @@
 #ifndef OPENTHREAD_COMMISSIONER_H_
 #define OPENTHREAD_COMMISSIONER_H_
 
+#ifdef OTDLL
+#ifndef OTAPI
+#define OTAPI __declspec(dllimport)
```

Comment:
Can we unify these in a single location, such as `include/platform/toolchain.h`?